language_bindings/rust/rust_tests/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+//!DO NOT PUBLISH
```

Comment:
Specify this explicitly in the Cargo.toml instead: https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field